deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: oadp-operator
           # Replace this with the built image name
-          image: "quay.io/konveyor/oadp-operator:latest"
+          image: "quay.io/deshah/oadp:latest"
```

Comment:
Please remove this change.

Reply:
These are operator-courier changes, it pulls the image from here which is pushed using operator-courier by specifying metadata versions. So should I push it to konveyor/oadp-operator?

Reply:
I don't think operator-courier is making this change. This is what you change when you build your own operator. People use this file to deploy the upstream version of this operator it needs to point to Konveyor.